000-debug/debug-mode.php

```diff
@@ -84,32 +85,33 @@ function disable_debug_mode() {
 	nocache_headers();
 
 	$ttl = time() - COOKIE_TTL;
-	setcookie( 'vip-go-cb', '', $ttl );
-	setcookie( 'a8c-debug', '', $ttl );
+	setcookie( 'vip-go-cb', '', $ttl );     // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.cookies_setcookie
+	setcookie( 'a8c-debug', '', $ttl );     // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.cookies_setcookie
 
 	send_pixel( [ 'vip-go-a8c-debug' => 'disable' ] );
 
 	redirect_back();
 }
 
 function toggle_debug_mode() {
+	// phpcs:disable WordPress.Security.ValidatedSanitizedInput.InputNotValidated -- we check if `a8c-debug` exists in `init_debug_mode`
+	// phpcs:disable WordPress.Security.NonceVerification.Recommended
 	if ( 'true' === $_GET['a8c-debug'] ) {
 		enable_debug_mode();
-		return;
-    } elseif ( 'false' === $_GET['a8c-debug'] ) {
+	} elseif ( 'false' === $_GET['a8c-debug'] ) {
 		disable_debug_mode();
-		return;
 	}
+	// phpcs:enable
 }
```

Comment:
An idea for a future PR: rename this function to `maybe_toggle_debug_mode` and move the `isset()` check from [`init_debug_mode()`](https://github.com/Automattic/vip-go-mu-plugins/pull/2468/files#diff-5b9a1e0cf799a444a8901278aa96878ada5653f426f244ba13582db47583cddfR27) here. This will let us avoid unnecessary suppression and will make potential refactoring easier.

Reply:
If there are no objections, I can do that.